test.py

```diff
@@ -409,6 +409,24 @@ def pattern(self) -> str:
         return "*test.cql"
 
 
+class TopologyTestSuite(PythonTestSuite):
```

Comment:
We have a topology/ subdirectory, why can't the extra logic for topology tests be there, and not in the main test.py?
I think we're putting too much code in test.py, and there is a lot of duplication in test.py and it makes it harder to fix things or to understand which code controls what.
But I won't insist, I understand that we have different philosophies regarding test.py.

Reply:
Moreover, the comment below doesn't help me understand how "tests dealing with topology changes" are different from other suite types and why we need a separate test suite type for it. Is the difference that these tests run on multi-node clusters and not one node? Is the difference that these tests each *changes* the cluster, so we need a new cluster for each test? Or something else?